fr-fr/php.html.markdown

```diff
@@ -148,6 +148,35 @@ $array[] = 'Four';
 // Retrait d'un élément du tableau
 unset($array[3]);
 
+//Parcourir un tableau
+for($i=0;$i<sizeof($array);$i++)
```

Comment:
I'd merge this, but:
 - in PHP, count() is preferred to sizeof()
 - foreach is usually the recommended way